dragonfly_doe2/doe/floor_space.py

```diff
@@ -92,6 +93,65 @@ def __repr__(self):
         return self.to_inp()
 
 
+@dataclass
+class Window:
+    """*.inp Window object.
+
+    Init methods(s):
+        1. from_window(name, x, y, width, height)
+
+    """
+    name: str
+    location: int
+    x: float
+    y: float
+    width: float
+    height: float
+
+    @classmethod
+    def from_window(cls, nm, x, y, w, h):
+        return cls(name=nm, x=x, y=y, width=w, height=h)
+
+    def to_inp(self):
+        return f'{self.name}_wall{self.location} = WINDOW\n   ' \
+               f'X           = {self.x}\n   ' \
+               f'Y           = {self.y}\n   ' \
+               f'WIDTH       = {self.width}\n   ' \
+               f'HEIGHT      = {self.height}\n   '\
+               f'GLASS-TYPE  = WT1\n   ..\n'
+
+    def __repr__(self):
+        return self.to_inp()
+
+
+@dataclass
+class WindowSet:
+    name: str
+    location: int
+    window_data: List[Window]
```

Comment:
what about windows for a list of windows?

```suggestion
    windows: List[Window]
```